src/ir_Mitsubishi.cpp

```diff
@@ -85,6 +88,54 @@ void IRsend::sendMitsubishi(uint64_t data, uint16_t nbits, uint16_t repeat) {
 }
 #endif   // SEND_MITSUBISHI
 
+#if DECODE_MITSUBISHI_AC
+// Decode a byte from the marks and spaces.
+//
+// Args:
+//   resultByte:  the pointer for the result.
+//   data_ptr:    pointer for the input.
+//   mark:        desired length of the mark.
+//   onespace:    desired length of the space for 1.
+//   zerospace:   desired length os the space for 0.
+//   MSBFirst:    most significant bit first? (default false).
+//
+// Status: DEAT / Appears to be working.
+//
+bool IRrecv::matchByte(uint8_t *resultByte, volatile uint16_t *data_ptr,
```

Comment:
We should probably make `matchData()` handle LSB data rather than add another routine.

Reply:
You are right. I didn't want to modify such a widely used function for my needs. This is a more light weight solution but if you prefer I can move this functionality to there.

Reply:
I think it's best if we can. I use it elsewhere and then do `reverseBits()` when I want it in LSB order. Doing it as part of the match is a better idea.

Don't worry about breaking it. That's what the unit tests are for. They'll catch most things if you break it. ;)

Reply:
I've upgraded `matchData()` to handle LSBF bit ordering. So this function can go and convert to use the standard `matchData()` function.